app/views/govuk_publishing_components/components/_title.html.erb

```diff
@@ -1,6 +1,7 @@
 <%
   average_title_length ||= false
   context ||= false
+  context_locale ||= false
```

Comment:
This is currently setting `lang="false"` if no context_locale is passed. Perhaps only add the lang attribute if this isn't false?